js/src/main/scala/crystal/react/StreamRenderer.scala

```diff
@@ -9,35 +9,40 @@ import japgolly.scalajs.react.vdom.html_<^._
 import _root_.io.chrisdavenport.log4cats.Logger
 
 import scala.scalajs.js
+import crystal.data._
+import crystal.data.implicits._
+import crystal.data.react.implicits._
 
 object StreamRenderer {
-  type Props[A]     = A => VdomNode
+  type Props[A]     = Pot[A] => VdomNode
   type Component[A] =
     CtorType.Props[Props[A], UnmountedWithRoot[Props[A], _, _, _]]
 
-  type State[A] = Option[A]
+  type State[A] = Pot[A]
 
   def build[F[_]: ConcurrentEffect: Logger, A](
     stream:      fs2.Stream[F, A],
     reusability: Reusability[A] = Reusability.by_==[A],
     key:         js.UndefOr[js.Any] = js.undefined
   ): Component[A] = {
-    implicit val propsReuse: Reusability[Props[A]] = Reusability.byRef
-    implicit val aReuse: Reusability[A]            = reusability
+    implicit val propsReuse: Reusability[Props[A]] =
+      Reusability.byRef // Should this be Reusable[Props[A]]?
```

Comment:
Sounds like it should be `Reusability.by[A]` or something and take `Reusability[A]` as an implict?

Reply:
The thing is that here `Props[A]` is a function `Pot[A] => VdomNode`. So we are back to the decision of whether allowing anonymous functions in clients or forcing them to declare them as stable with `Reusable`.

Now that I have a 2nd look at it, `byRef` makes no sense, since it's the worst of both worlds. I'll add a parameter for `Reusability[Pot[A] => VdomNode]` but use `Reusability.always` by default, assuming the function doesn't change but allowing override when needed.